addons/website_event_track/models/event_track_stage.py

```diff
@@ -19,15 +19,28 @@ class TrackStage(models.Model):
     fold = fields.Boolean(
         string='Folded in Kanban',
         help='This stage is folded in the kanban view when there are no records in that stage to display.')
-    is_accepted = fields.Boolean(
-        string='Accepted Stage',
-        help='Accepted tracks are displayed in agenda views but not accessible.')
-    is_done = fields.Boolean(
-        string='Done Stage',
-        help='Done tracks are automatically published so that they are available in frontend.')
+    is_visible_in_agenda = fields.Boolean(string='Visible in agenda',
+        compute='_compute_is_visible_in_agenda',
+        help='If checked, the element will be visible in the frontend',
+        store=True)
+    is_fully_accessible = fields.Boolean(string='Fully accessible',
```

Comment:
``is_page_accessible`` ? That way we indicate this gives access to the page view. I don't like the 'fully accessible' as it is quite vague, especially a not published record is not "accessible". Not a fan of "page accessible", but no better naming currently ... ``is_validated`` maybe, telling record is complete and page view is accessible if rights allow it ?